app/src/main/java/com/kamron/pogoiv/OCRHelper.java

```diff
@@ -169,7 +169,6 @@ public int getPokemonEvolutionCostFromImg(Bitmap pokemonImage) {
             }
             ocrCache.put(hash, stringCacheEvoCandyCost);
         }
-        System.out.println("asdasdasd");
```

Comment:
It's fine to have these commits in your local branch, but when you think it's good enough for a PR, please squash these "local mistakes".